### ISSUE TITLE
Update fsa-field examples and document if necessary.

### ISSUE THREAD

**Issue by francisrupert** [USDA-FSA/fsa-design-system#156]
Initially **BLOCKED** by https://github.com/USDA-FSA/fsa-style/issues/222

`fsa-field` examples on http://usda-fsa.github.io/fsa-design-system/components/form-fields/ are out-of-sync with `fsa-style`-equivalent.

Most notably, they're missing the added `aria` attributes. Note the use of `aria-describedby="[id-value]"` and `role="alert"` below...

```html
<div class="fsa-field fsa-field--error">
  <label class="fsa-field__label" for="lorem-7650">Error label <span class="fsa-field__label-desc">Required</span></label>
  <input class="fsa-input fsa-field__item fsa-input--error" aria-describedby="lorem-7650--message" id="lorem-7650" name="lorem-7650" type="text" value="">
  <span class="fsa-field__help">This is how people can contact you</span>
  <span class="fsa-field__message" id="lorem-7650--message" role="alert">Helpful error message</span>
</div>
```

**Comment by bignamehere** [comment]
@francisrupert can this be worked on, or is it still blocked? I can compile the Design System now, so I can work on this. let me know.

**Comment by bignamehere** [comment]
@francisrupert  I could use a second set of eyes on this Issue. Let me know if you see any of the ARIA that wasn't implemented correctly, or if we need to add something more.

2def74a

**Comment by francisrupert** [comment]
@bignamehere,

See the commits above that amends this Issue (156). I'm pretty confident in it, but a second set of eyes back at what I did is warranted.

Both of our changes are on `gh-pages` branch, and thus already published on the public site.

**Comment by bignamehere** [comment]
@francisrupert 

I looked it over, and everything appears to be good.
